SerialPrograms/Resources/PokemonSV/AAT/TournamentPrizeList.json

```diff
@@ -0,0 +1,67 @@
+[
+    "beast-ball",
+    "dream-ball",
+    "fast-ball",
+    "friend-ball",
+    "heavy-ball",
+    "level-ball",
+    "love-ball",
+    "lure-ball",
+    "moon-ball",
+    "bottle-cap",
+    "gold-bottle-cap",
+    "rare-candy",
+    "nugget",
+    "big-nugget",
+    "pp-up",
+    "pp-max",
+    "hp-up",
+    "protein",
+    "iron",
+    "calcium",
+    "zinc",
+    "carbos",
+    "adamant-mint",
+    "bashful-mint",
+    "bold-mint",
+    "brave-mint",
+    "calm-mint",
+    "careful-mint",
+    "docile-mint",
+    "gentle-mint",
+    "hardy-mint",
+    "hasty-mint",
+    "impish-mint",
+    "jolly-mint",
+    "lax-mint",
+    "lonely-mint",
+    "mild-mint",
+    "modest-mint",
+    "naive-mint",
+    "naughty-mint",
+    "quiet-mint",
+    "quirky-mint",
+    "rash-mint",
+    "relaxed-mint",
+    "sassy-mint",
+    "serious-mint",
+    "timid-mint",
+    "normal",
+    "fire",
+    "water",
+    "electric",
+    "grass",
```

Comment:
Wait, are these items? lol

Reply:
Tera shards. Couldn't find translations for them so I put the types instead. The filter dropdown has tera shard written in it though.
![image](https://user-images.githubusercontent.com/3212217/217380856-3994a431-dada-4c46-bdcc-ffd1907a2114.png)